model/patch_lifecycle.go

```diff
@@ -676,6 +665,27 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	return patchVersion, nil
 }
 
+// mergeWithBaseManifest merges the manifest from the patch with the manifest from the
+// most recent base commit.
+func mergeWithBaseManifest(mfst *manifest.Manifest, patchVersion *Version) error {
+	if mfst != nil {
+		baseManifest, err := manifest.FindFromVersion(patchVersion.Id, patchVersion.Identifier, patchVersion.Revision, patchVersion.Requester)
+		if err != nil {
+			return errors.Wrap(err, "constructing base manifest")
+		}
+		if baseManifest != nil {
+			modulesToCopy := baseManifest.Modules
+			for key := range modulesToCopy {
+				if _, ok := mfst.Modules[key]; ok {
+					modulesToCopy[key] = mfst.Modules[key]
```

Comment:
I might be forgetting what's going on now but I thought that we'd want to use the base manifest only for things that aren't auto-update (i.e. exists in the manifest), so !ok

Reply:
That's what should be happening here - we initialize `modulesToCopy` as the base manifest modules and for anything we find in `mfst` (which contains only auto-update modules) we replace the value in `modulesToCopy`.

Reply:
Not relevant anymore since I'm deleting this function though